charts/vector/docs/quickstart.md

```diff
@@ -0,0 +1,168 @@
+By default the Agents will forward their logs and metrics directly to Datadog's
+hosted intake, we'll create a values file to override that behavior to forward to
+your new Vector Aggregators.
+
+For Datadog Agents version `7.35`/`6.35` or greater:
+
+```bash
+cat <<-'VALUES' > dd-agent.yaml
+datadog:
+  containerExclude: "name:vector"
+  logs:
+    enabled: true
+    containerCollectAll: true
+agents:
+  useConfigMap: true
+  customAgentConfig:
+    kubelet_tls_verify: false
+    vector:
+      logs:
+        enabled: true
+        url: http://vector-haproxy.vector:8282 # Use https if SSL is enabled in Vector source configuration
+      metrics:
+        enabled: true
+        url: http://vector-haproxy.vector:8282 # Use https if SSL is enabled in Vector source configuration
+VALUES
+```
+
+For Datadog Agents version `7.34`/`6.34` and older:
+
+_NOTE: Metrics will not be routed through Vector for Datadog Agent versions older
+than `7.35`/`6.35`._
+
+```bash
+cat <<-'VALUES' > dd-agent.yaml
+datadog:
+  containerExclude: "name:vector"
+  logs:
+    enabled: true
+    containerCollectAll: true
+agents:
+  useConfigMap: true
+  customAgentConfig:
+    kubelet_tls_verify: false
+    logs_config:
+      logs_dd_url: vector-haproxy.vector:8282
+      logs_no_ssl: true # Set to false if SSL is enabled in Vector source configuration
+      use_http: true
+VALUES
+```
+
+**WARNING:** The above config disables TLS verification of the connection to the
+Kubelet. This is useful in a quickstart environment like `minikube`, but when
+real certificates are available, the `kubelet_tls_verify` option should be removed.
+
+_NOTE: When you have the `datadog_agent` source configured, a `helm install` or
+`helm upgrade` will output the appropriate values to provide to the datadog helm
+chart._
+
+```bash
+helm install datadog datadog/datadog --namespace datadog --create-namespace \
+	--values dd-agent.yaml --set datadog.apiKey="${DATADOG_API_KEY}"
+```
+
+Once the Datadog Agents are running you should be able to see your logs in
+Datadog's [Live Tail](https://app.datadoghq.com/logs/livetail?query=sender%3Avector) view,
+by filtering for `sender:vector`. You can also use `vector top` to view Vector's
+topology and related metrics:
+
+```bash
+kubectl --namespace vector exec --stdin --tty statefulset/vector -- vector top
+```
+
+### Troubleshooting
+
+To diagnose problems the following commands may prove useful:
+
+For Datadog Agents:
+
+```bash
+kubectl --namespace datadog exec --stdin --tty daemonset/datadog -- agent status
+```
+
+For the HAProxy:
+
+```bash
+kubectl --namespace vector logs --follow deployment/vector-haproxy
+```
+
+For the Vector Aggregators:
+
+```bash
+kubectl --namespace vector exec --stdin --tty statefulset/vector -- vector tap internal_logs
+```
+
+## Moving to the cloud
+
+### Elastic Kubernetes Service
+
+#### Prerequisites
+
+#### Installing
+
+#### Troubleshooting
```

Comment:
Might link out to separate docs for each cloud provider rather than have everything in one file...